### DISCUSSION TITLE
Updating Boilerplate.AspNetCore to 2.2.3 causes missing ITranslator<,>

### DISCUSSION THREAD

**Discussion by andre-artus** [Dotnet-Boxed/Templates#172] [Q&A]
I was notified about an update to Boilerplate.AspNetCore, so I updated to 2.2.3 from 2.2.2. This results in a slew of error messages when I open the affected project, and the project fails to build. When I revert to 2.2.2 it seems fine.

Is there something missing, or am I missing something? Going through the git repo I still see numerous references to ITranslator.

As an aside, how do I update the template for "dotnet new"? I thought maybe the template changed and so I wanted to update it before creating a new test site, but I could not find how. My templates is still 1.1.2.

**Comment by RehanSaeed** [comment]
The translation logic was moved into it's own NuGet package `Boilerplate.Translation`. I was going to update the template and add a note about it in the release notes.
